packages/postgres-database/docker/Dockerfile

```diff
@@ -16,10 +16,8 @@ FROM base as build
 
 RUN apt-get update \
   && apt-get install -y --no-install-recommends \
-    build-essential \
-    git \
-  && apt-get clean \
-  && rm -rf /var/lib/apt/lists/*
```

Comment:
This is in the build part of the Dockerfile, which is disregarded anyway in the end since it is a multi-stage Dockerfile.
Also caching will not work if the cache files are deleted in the RUN clause.
It makes sense to clean the cache if the we are in the production stage where we will spare quite some space.